tools/ci_build/github/linux/ort_minimal/build_full_ort_and_create_ort_files.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# This script will run a full ORT build and use the python package built to generate ort format test files,
+# and the exclude ops config file, which will be used in the build_minimal_ort_and_run_tests.sh
+
+set -e
+
+# Run a full build of ORT
+# Since we need the ORT python package to generate the ORT format files and the include ops config files
+# Will not run tests since those are covered by other CIs
+/opt/python/cp37-cp37m/bin/python3 /onnxruntime_src/tools/ci_build/build.py \
+    --build_dir /build --cmake_generator Ninja \
+    --config Debug \
+    --skip_submodule_sync \
+    --parallel \
+    --build_wheel \
+    --skip_tests \
+    --enable_pybind \
+    --cmake_extra_defines PYTHON_INCLUDE_DIR=/opt/python/cp37-cp37m/include/python3.7m PYTHON_LIBRARY=/usr/lib64/librt.so
+
+# Install the ORT python wheel
+/opt/python/cp37-cp37m/bin/python3 -m pip install -U /build/Debug/dist/*
+
```

Comment:
Uninstall this once done converting just so there's no chance something downstream accidentally uses the full version from the package